server/games/ladder_game.py

```diff
@@ -34,3 +36,18 @@ def get_army_score(self, army: int) -> int:
         as 1 for win and 0 for anything else.
         """
         return self._results.victory_only_score(army)
+
+    def _outcome_override_hook(self) -> Optional[List[GameOutcome]]:
+        if not config.LADDER_1V1_OUTCOME_OVERRIDE or len(self.players) > 2:
+            return None
+        team_sets = self.get_team_sets()
+        army_scores = [
+            self._results.score(self.get_player_option(team_set.pop().id, "Army"))
```

Comment:
Cannot use `self.get_army_score(...)`, since that was overridden to use the reported result instead.
I thought the better way was
```suggestion
            super().get_army_score(self.get_player_option(team_set.pop().id, "Army"))
```
but for some reason the `super()` call fails, saying that the types aren't related?
```
super(type, obj): obj must be an instance or subtype of type
```
Forcing it's hand with `super(Game, self)` says that it has no method `get_army_score`, I was very confused and still don't know what was going on.

Reply:
I think the right way to explicitly call it would be `super(LadderGame, self)`, the docstring for super says:
```
super() -> same as super(__class__, <first argument>)
```
But I'm not gonna worry about investigating it right now :P